src/DS/rds.c

```diff
@@ -730,6 +732,36 @@ static int ds_rds_create_from_dom(xmlDocPtr* ret, xmlDocPtr sds_doc, xmlDocPtr x
 	xmlDOMWrapReconcileNamespaces(sds_wrap_ctxt, sds_res_node, 0);
 	xmlDOMWrapFreeCtxt(sds_wrap_ctxt);
 
+	if (tailoring_doc) {
+		const char *tailoring_component_id = "scap_org.open-scap_comp_user_file_tailoring";
+		const char *tailoring_component_ref_id = "scap_org.open-scap_cref_user_file_tailoring";
```

Comment:
For reference, this needs to be added somewhere here:
- ensure that the tailoring component IDs are unique.
- Make a function `get_unique_id(DOM, starting_id)`

Reply:
@matejak We already have it at other place. Take a look here:
https://github.com/OpenSCAP/openscap/blob/166a0a32fc9f72dc5aa9bddba164af8066fce724/src/DS/sds.c#L1059-L1079